lib/commonConfig.js

```diff
@@ -130,7 +130,7 @@ function processEnvironmentVariables() {
             'IOTA_IOTAM_PROTOCOL',
```

Comment:
The fix included in this PR should be mentioned in CHANGES_NEXT_RELEASE. Suggestion:

```
- Fix: IOTA_EXPLICIT_ATTRS env var was not working
```

Reply:
Fix 0ab2b88